pygmt/tests/test_grdfilter.py

```diff
@@ -0,0 +1,55 @@
+"""
+Tests for grdfilter.
+"""
+import numpy as np
+import pytest
+import xarray as xr
+from pygmt import grdfilter, grdinfo
+from pygmt.datasets import load_earth_relief
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import GMTTempFile
+
+
+@pytest.fixture(scope="module", name="grid")
+def fixture_grid():
+    """
+    Load the grid data from the sample earth_relief file.
+    """
+    return load_earth_relief(registration="pixel")
+
+
+def test_grfilter_dataarray_in_dataarray_out(grid):
+    """
+    grdfilter an input DataArray, and output as DataArray.
+    """
+    # grid = load_earth_relief(registration="pixel")
```

Comment:
```suggestion
```

Reply:
Thanks, forgot to remove it after loading the pytest fixture